scripts/avocado

```diff
@@ -56,10 +56,6 @@ def handle_exception(*exc_info):
     sys.exit(-1)
 
 
-if __name__ == '__main__':
-    sys.excepthook = handle_exception
```

Comment:
Then the avocado module import can be moved as well (it doesn't affect style rules/checks).